radiance.go

```diff
@@ -312,6 +381,8 @@ type lazyDialingRoundTripper struct {
 var _ http.RoundTripper = (*lazyDialingRoundTripper)(nil)
 
 func (lz *lazyDialingRoundTripper) RoundTrip(req *http.Request) (*http.Response, error) {
+	ctx, span := tracer.Start(req.Context(), "lazy-dialing-roundtrip")
```

Comment:
We could add a error field to this span at line 393. We could also receive the error and response from the roundtrip and add some fields like the HTTP status code and the error